src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public abstract class Figure implements AreaCalculable, Drawable {
```

Comment:
Consider whether you need `Figure` to be an abstract class. Abstract classes are used to provide a common definition of a base class that multiple derived classes can share. However, if you're setting behavior (methods) that all subclasses should implement, an interface might be more appropriate.